.gitignore

```diff
@@ -8,3 +8,4 @@ docs/.vuepress/public/assets/json/packages.json
 /packages/manager/tools/testcafe/config/**/*.json
 /reports
 .eslintcache
+/.idea/
```

Comment:
`.idea` folder is specific to JetBrain editors, and we prefer to avoid adding editor specific files in `.gitignore`.
if you want to ignore it for all of your repository, you could define a global `gitignore` file as explained in https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer .
Thanks.